examples/benchmark/requirements-benchmark.txt

```diff
@@ -0,0 +1,14 @@
+# Evaluation
+numpy>=1.20.0
+pandas>=1.3.1
+plotly>=5.1.0
+pyyaml>=5.4.1
+sleepecg>=0.2.0
+
```

Comment:
Exporting figures requires `kaleido`, so this package should be added here (not sure about the min version).